test/python/test_api.py

```diff
@@ -38,10 +38,6 @@ def test_enable(self):
     def test_backends_len(self):
         assert ngraph_bridge.backends_len()
 
-    def test_set_backend(self):
```

Comment:
This test sets backend. and does not reset it. We should not have tests setting the backend as it would affect the entire suite. The functionality has been tested in C++ tests. So we dont need it